packages/pages/package.json

```diff
@@ -90,7 +90,7 @@
     "@types/express-serve-static-core": "^4.17.29",
     "@types/fs-extra": "^9.0.13",
     "@types/glob": "^7.2.0",
-    "@types/jest": "^27.5.2",
+    "@types/jest": "^27.5.1",
```

Comment:
You actually moved the version down which seems weird, though it doesn't actually matter.

Reply:
It was at 27.5.1 before. I thought. the check failure was caused by this so gave it a try. 27.5.1 should be fine to use